app/tournament/battle_cls.py

```diff
@@ -0,0 +1,64 @@
+from time import sleep
+
+from app.knights.knight_cls import Knight
+from app.knights.knights_bases import KNIGHTS
+from app.tournament.battle_funcs import Battle
+from app.tournament.battle_texts import GREETING, TOURNAMENT_BEGIN, \
+    REMOVED_KNIGHT, TOURNAMENT_CANCEL
+
+
+class Tournament:
+    def __init__(self, knights: list[Knight]) -> None:
+        self.knights = knights
+
+    def __print_knights__(self, pop_knight_name="") -> None:
```

Comment:
Please explain why you always add __ to the end of function?
Read about magic, private, protected and public methods